### DISCUSSION TITLE
KittyCats

### DISCUSSION THREAD

**Discussion by AcelyaP** [sonsuzus/BilsemPython#69] [Genel]
Merhaba benim adım Açelya Pirtini ve ben bu takımı kuruyorum. Katılmak isteyenler isminizi ,grubunuzu vb. şeyleri paylaşabilir.
Öyg 1 öğrencisiyim cumartesi günleri 3 ve 4. derslere giriyorum.(Katılanlar WhatsApp grubuna katılırlarsa sevinirim)

**Comment by NidaZeynep** [comment]
merhaba takımına katılmak istiyorum ama nasıl yapacağımızı gerçekten anlamadım. bende seninle aynı gün aynı dersleri alıyorum. yardımcı olabilirsen sevinirim

**Reply by AcelyaP** [reply]
Ben WhatsAppdan bir grub kuracağım 0532 390 36 30 numarasına mesaj atarsan sevinirim

**Comment by eyupbjk** [comment]
merhaba ben Eyüp Ensar Çetin cuma 1 ve 2. derslere giriyorum katılmak istiyorum

**Reply by AcelyaP** [reply]
Takima hoş geldin !Ben WhatsAppdan bir grub kurdum 0532 390 36 30 numarasına mesaj atarsan sevinirim.

**Comment by Barlasgng** [comment]
merhaba ben barlas takımına bende katılmak istiyorum çarşamba günü ilk 2 dersteyim

**Reply by AcelyaP** [reply]
Takima hoş geldin Barlas !WhatsApp grubuna katılmak için 0532 390 36 30 numarasına mesaj atarsan sevinirim.

**Comment by slos8888** [comment]
Ezel Kocaaslan ÖYG 1 3. ve 4. derslere giriyorum ve bu takıma katılmak istiyorum

**Reply by AcelyaP** [reply]
Merhaba Ezel takima hoş geldin!Ben WhatsAppdan bir grub kurdum katılmak için 0532 390 36 30 numarasına mesaj atarsan sevinirim ☺️
